wgpu/src/shader/vertex.wgsl

```diff
@@ -0,0 +1,7 @@
+// Compute the normalized quad coordinates based on the vertex index.
+fn vertex_position(vertex_index: u32) -> vec2<f32> {
+    // #: 0 1 2 3 4 5
+    // x: 1 1 0 0 0 1
+    // y: 1 0 0 0 1 1
+    return vec2<f32>((vec2(1u, 2u) + vertex_index) % 6u < 3u);
```

Comment:
Cool bit of code here! Didn't think about calculating a vec2<bool> and letting it be coerced to 1.0/0.0.